src/Autofac.Integration.Mvc/Autofac.Integration.Mvc.csproj

```diff
@@ -1,164 +1,76 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProjectGuid>{DD874E64-C7EC-464D-925F-CF4A709EDEEF}</ProjectGuid>
-    <OutputType>Library</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>Autofac.Integration.Mvc</RootNamespace>
+    <Description>Autofac MVC Integration</Description>
+    <!-- VersionPrefix patched by AppVeyor -->
+    <VersionPrefix>0.0.1</VersionPrefix>
+    <TargetFramework>net461</TargetFramework>
```

Comment:
Should we stick with net461 or take the opportunity to go, like, net472?

Reply:
I'm not sure we need to? We don't really gain anything from it, because we'll need to support netstandard20 on Autofac as long as we want to support any of the .NET Framework versions, so all we'd be doing is punishing those people who sadly are stuck on net461.

Reply:
Only reason I raised it was that weird thing where net461 is _mostly_ but not _totally_ netstandard2 compliant. [Docs here.](https://docs.microsoft.com/en-us/dotnet/standard/net-standard)

> While NuGet considers .NET Framework 4.6.1 as supporting .NET Standard 1.5 through 2.0, there are several issues with consuming .NET Standard libraries that were built for those versions from .NET Framework 4.6.1 projects. For .NET Framework projects that need to use such libraries, we recommend that you upgrade the project to target .NET Framework 4.7.2 or higher.

That is, this library is consuming a netstandard2 library and I'm afraid by leaving it net461 there's a potential for some weird transient failure to occur.

Reply:
Yeah; apparently [I already had this discussion with myself in the Autofac PR](https://github.com/autofac/Autofac/pull/1161#issuecomment-653218762), so let's make it net472.